uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/API/Query/LeaveRequestSelect.php

```diff
@@ -86,17 +89,27 @@ private function addWhere(CRM_Utils_SQL_Select $customQuery) {
         )'
     ];
 
+    if($hasUnassignedAsFalse) {
+      $conditions[] = implode(' AND ', $this->hasActiveLeaveManagerCondition());
```

Comment:
can't we use array_merge instead and leave the concatenation job to civi (That is, we just pass an array of conditions and let civi do the job)?